### ISSUE TITLE
errors when loading project in VS 2019

### ISSUE THREAD

**Issue by petrosonic** [csharpschool/VideoOnDemand22#1]
HI, downloaded this project and when trying to load the project in VS2019 a lot of errors are showing up. 
Screenshot 1:
![error1](https://user-images.githubusercontent.com/37005323/58947769-eb084300-8788-11e9-9d7c-bbdda7010881.JPG)

Tried to add missing references but getting error again - pic2.
![error2](https://user-images.githubusercontent.com/37005323/58947780-efccf700-8788-11e9-9a2c-d0ae815d2d46.JPG)

Stack trace is:
System.AggregateException : One or more errors occurred. ---> Unable to find a type of reference that is appropriate for this file: "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.extensions.identity.stores\2.2.0\lib\netstandard2.0\Microsoft.Extensions.Identity.Stores.dll".
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.InvokeWorker(Func`1 getFromDocument,IProgressTracker progressTracker,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.<>c__DisplayClass17_0.<InvokeCore>b__0()
   at Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformAction(IExtensionManager extensionManager,Object extension,Action action)
---> (Inner Exception #0) System.ArgumentException : Unable to find a type of reference that is appropriate for this file: "C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.extensions.identity.stores\2.2.0\lib\netstandard2.0\Microsoft.Extensions.Identity.Stores.dll".
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.Add(String bstrPath)
   at Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.VisualStudioWorkspaceImpl.ApplyMetadataReferenceAdded(ProjectId projectId,MetadataReference metadataReference)
   at Microsoft.CodeAnalysis.Workspace.ApplyProjectChanges(ProjectChanges projectChanges)
   at Microsoft.CodeAnalysis.Workspace.TryApplyChanges(Solution newSolution,IProgressTracker progressTracker)
   at Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.VisualStudioWorkspaceImpl.TryApplyChanges(Solution newSolution,IProgressTracker progressTracker)
   at Microsoft.CodeAnalysis.CodeActions.ApplyChangesOperation.TryApply(Workspace workspace,IProgressTracker progressTracker,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Editor.Implementation.CodeActions.CodeActionEditHandlerService.ProcessOperations(Workspace workspace,ImmutableArray`1 operations,IProgressTracker progressTracker,CancellationToken cancellationToken)
   at async Microsoft.CodeAnalysis.Editor.Implementation.CodeActions.CodeActionEditHandlerService.ApplyAsync(<Unknown Parameters>)
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)<---

Any suggestion on steps to be taken when loading this project from scratch in a new VS2019 env?
Thanks

**Comment by ondrovic** [comment]
I had no issue with this loading in VS2019 maybe try doing a right click on the solution and doing a clean and rebuild?
![image](https://user-images.githubusercontent.com/3682042/59103728-48b99e00-88f5-11e9-9c57-f4b268816a70.png)
![image](https://user-images.githubusercontent.com/3682042/59103751-53743300-88f5-11e9-85e1-82fd46b5eab8.png)

**Comment by reshea** [comment]
When I went to create the database, the "SqlServerValueGenerationStrategy" was reporting as not existing in the 'Migrations'-->".cs" files.  I added the enum: 
public enum SqlServerValueGenerationStrategy { SequenceHiLo = 0, IdentityColumn = 1  }
and the build of the database through the Package Manager proceed OK.  
  I am using Visual Studio Community 2019. 
I was just happy to get the database created for now.  I used a SQL Server DB I have on my NAS (Win2016 server [I thought I would 'challenge' myself - in addition to the book].

I have just started today.  However, I do like the writing style.  The above is more an FYI and maybe I did something wrong in the setup of things.
